endpoints/openrtb2/auction.go

```diff
@@ -2352,6 +2352,33 @@ func setDoNotTrackImplicitly(httpReq *http.Request, r *openrtb_ext.RequestWrappe
 	}
 }
 
+func secCookieDeprecation(httpReq *http.Request, r *openrtb_ext.RequestWrapper) {
```

Comment:
There are some requirements in the spec not represented in this code:

> /auction endpoint
>
> 3] If the account config auction.privacysandbox.cookiedeprecation is false or not specified, we're done - nothing else to do here.
>
> 5] If the value is a string that's less than 100 characters, copy it to $.device.ext.cdep. If it's greater than 100 chars, add a debug warning.

Reply:
Had to refactor `amp` and `video` endpoints to align with flow of `auction` so that `setFieldsImplicitly()` gets account object

Reply:
refer https://github.com/prebid/prebid-server/pull/3381#discussion_r1474857671